x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/CsvSpecTestCase.java

```diff
@@ -43,9 +43,7 @@ public CsvSpecTestCase(String fileName, String groupName, String testName, Integ
 
     @Override
     protected final void doTest() throws Throwable {
-        // Run the time tests always in UTC
-        // TODO: https://github.com/elastic/elasticsearch/issues/40779
```

Comment:
If we don't (yet) randomise the timezones for the QA tests ref'd in this comment (which I think is the case), maybe we should keep the comment. (BTW, the file ref'd in the issue contains a similar comment.)

Reply:
Maybe I got this wrong but when the comment was added, `time.csv-spec` has been special cased (see https://github.com/elastic/elasticsearch/pull/40776/files#diff-387c1ea94dd5d111032f384770a8244f7d4942ac1fd91a68b397614212d59f04R44). But the special case has been removed in the meantime and I think the comment is no longer accurate. Or the reality is now more complicated as described in https://github.com/elastic/elasticsearch/pull/76638/files#diff-aceee053cb68e1bb219cfe4256635c7f77d86b46eb943e5ad42bc7bb360ca945R125